src/machines/modelingMachine.ts

```diff
@@ -578,7 +581,9 @@ export const modelingMachine = setup({
     'has valid selection for deletion': () => false,
     'is-error-free': () => false,
     'no kcl errors': () => {
-      return !kclManager.hasErrors()
+      const hasErrors = kclManager.hasErrors()
+      if (hasErrors) toast.error(KCL_ERRORS_MESSAGE)
```

Comment:
Toasting in here is probably anti-pattern, but this hasn't led to any side effect in my testing and I'm a little out of options here?